ros_ws/src/auto_race_pg/include/remote_keyboard.h

```diff
@@ -0,0 +1,56 @@
+#ifndef REMOTE_KEYBOARD_H
+#define REMOTE_KEYBOARD_H
+
+#include <ros/ros.h>
+
+#include <std_msgs/Float64.h>
+#include <std_msgs/Int64.h>
+
+#include <signal.h>
+#include <termios.h>
+#include <time.h>
+
+#define MODE "keyboard"
+
+#define TOPIC_SPEED "/set/speed"
+#define TOPIC_ANGLE "/set/position"
```

Comment:
Shouldn't this be "/set/angle"?
```suggestion
#define TOPIC_ANGLE "/set/angle"
```

Reply:
this is a parameter, will change it to js0